torch/_subclasses/meta_utils.py

```diff
@@ -146,7 +146,7 @@ def meta_tensor(self, t, shape_env=None):
 
         def sym(x):
             if make_symbolic:
-                return shape_env.create_symbol(x)
+                return shape_env.create_symintnode(shape_env.create_symbol(x))
```

Comment:
How was this working at all before?

Reply:
It wasn't :P

Reply:
But I guess we didn't have any tests where we converted an input with a non-zero storageoffset.